packages/backend/src/models/SlackAuthenticationModel.ts

```diff
@@ -101,7 +102,7 @@ export class SlackAuthenticationModel {
             )
             .select<(DbSlackAuthTokens & DbUser)[]>('*')
             .where('slack_team_id', teamId);
-        if (row === undefined) {
+        if (isNil(row) || isNil(row.user_uuid)) {
```

Comment:
This could probably be shortened with optional chaining: 

```suggestion
        if (isNil(row?.user_uuid)) {
```

Reply:
Thank you for your suggestion!